apps/api/src/server/applications/subscriptions/subscriptions.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * @typedef {import('../../../message-schemas/events/nsip-subscription.d.js').NSIPSubscription} NSIPSubscription
+ */
+
+/**
+ * @param {import('@prisma/client').Subscription} subscription
+ * @returns {NSIPSubscription}
+ */
+export function buildSubscriptionPayload(subscription) {
+	/** @type {NSIPSubscription} */
+	const payload = {
+		caseReference: subscription.caseReference,
+		emailAddress: subscription.emailAddress,
+		subscriptionType: subscription.subscriptionType
+	};
+	if (subscription.id) {
+		payload.subscriptionId = subscription.id;
+	}
+	if (subscription.startDate) {
+		payload.startDate = subscription.startDate.toISOString();
```

Comment:
JSON serialisation should do this for us (i.e. serialise dates to ISO strings)

Reply:
Because the JSON serialisation is implicit, I prefer to be explicit with the payload type. Should mean the tests are closer to the real setup too.